examples/ui/navigation/pubspec.yaml

```
@@ -0,0 +1,19 @@
+name: navigation_samples
+description: Sample navigation code.
+publish_to: none
+
+environment:
+  sdk: ^3.5.0
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  go_router: ^14.6.2
+
+dev_dependencies:
+  example_utils:
+    path: ../../example_utils
+
+flutter:
+  uses-material-design: true
```

We add an extra new line at the end of each file:

```suggestion
  uses-material-design: true

```